test/walkthrough-test.js

```diff
@@ -1,50 +0,0 @@
-/* global hoodie */
-module.exports = walktroughTest
-
-var logError = require('./utils/log-error')
-var toValue = require('./utils/to-value')
-
-function walktroughTest (test, selsa, server, debug) {
-  test('hoodie', function (t) {
-    t.test('walkthrough', function (tt) {
-      var username = 'user' + Math.random().toString(16).substr(2)
-
-      selsa.browser.url(server.info.uri).executeAsync(function storeAdd (username, done) {
-        return hoodie.store.add({
-          foo: 'bar'
-        }).then(done, done)
-      }, username).then(toValue).then(function (result) {
-        tt.is(result.foo, 'bar')
-      }).executeAsync(function signUp (username, done) {
-        return hoodie.account.signUp({
-          username: username,
-          password: 'secret'
-        }).then(done, done)
-      }, username).then(toValue).then(function (result) {
-        tt.is(result.username, username)
-      }).executeAsync(function signOut (username, done) {
-        return hoodie.account.signIn({
-          username: username,
-          password: 'secret'
-        }).then(done, done)
-      }, username).then(toValue).then(function (result) {
-        tt.is(result.username, username)
-      }).executeAsync(function signOut (done) {
-        return hoodie.account.signOut().then(done, done)
-      }).then(toValue).then(function (result) {
-        tt.is(result.username, username)
-      }).executeAsync(function storeFindAll (done) {
-        return hoodie.store.findAll().then(done, done)
-      }).then(toValue).then(function (result) {
-        tt.deepEqual(result, [])
-      }).catch(logError).then(function () {
-        tt.end()
-      })
-
-      // cleanup
-        .localStorage('DELETE')
-    })
-
-    t.end()
-  })
-}
```

Comment:
did you intentionally remove all these tests? I guess they are obsolete, just want to make sure

Reply:
Yes, I removed the tests - I think there are all redundant after switching to cypress?

Reply:
Yes, I thought so, too